### PR TITLE
Hotfix to export history count

### PR DESCRIPTION
**Pull request by debitan**
This is a follow up PR to https://github.com/uktrade/data-hub-frontend/pull/2454

I didn't think I could fix the count while waiting for the backend fix, but there is this slightly convoluted way to do it. 

This code is temporary and will be removed when the backend fix is ready.

### CODE REVIEW

File: src/apps/companies/apps/exports/tasks.js [hunk 1]
```diff
@@ -35,21 +35,27 @@ const createCountry = (item) => ({
...
+    results: cleanedResults.slice(0, 10).map(createCountry),
+  }
+}
 
 const handleError = (e) => Promise.reject(Error(e.response.data.detail))
 
-export const fetchExportsHistory = ({ companyId, activePage }) =>
-  axios
+export const fetchExportsHistory = ({ companyId, activePage }) => {
+  const offset = activePage * 10 - 10
```
Comment (adamledwards) [inline src/apps/companies/apps/exports/tasks.js]: Does offset always start from 1? 
If it is 0 is there a possibility you could send -10 to the server

File: src/apps/companies/apps/exports/tasks.js [hunk 2]
```diff
@@ -35,21 +35,27 @@ const createCountry = (item) => ({
...
+  return axios
     .post('/api-proxy/v4/search/export-country-history', {
       company: companyId,
-      limit: 10,
-      offset: activePage * 10 - 10,
+      offset,
```
Comment (adamledwards) [inline src/apps/companies/apps/exports/tasks.js]: With no limit is there a chance of duplicate data?
Sorry I'm not familiar with the whole implementation on data hub yet.

File: src/apps/companies/apps/exports/tasks.js (hunk 1 above)
Comment (debitan) [reply src/apps/companies/apps/exports/tasks.js]: it always starts at zero

File: src/apps/companies/apps/exports/tasks.js (hunk 2 above)
Comment (debitan) [reply src/apps/companies/apps/exports/tasks.js]: yeah, so will be throwing away any data beyond 10 items. It is inefficient, but the only way to calculate the correct total number of items while this bug exists.

**Pull request merged**